bin/main.rb

```diff
@@ -0,0 +1,77 @@
+hash = { 1 => 1, 2 => 2, 3 => 3, 4 => 4, 5 => 5, 6 => 6, 7 => 7, 8 => 8, 9 => 9 }
```

Comment:
[OPTIONAL]

Since you already have the main file this file is sort of redundant. It's not necessary.